packages/@lightningjs/ui-components/src/components/TabBar/Tab.styles.js

```diff
@@ -20,6 +20,7 @@ export const base = theme => ({
   radius: theme.radius.xl,
   paddingX: theme.spacer.xxxl + theme.spacer.xxs,
   paddingY: theme.spacer.md + theme.spacer.xs,
+  /** @deprecated */
```

Comment:
You don't need this in styles files, only TypeScript definition files. You 'll just update the `noTitlePaddingX` to `paddingXNoTitle`.

Reply:
Fixed